src/types/metaFactory.ts

```diff
@@ -64,5 +67,33 @@ export const MetaFactoryTypes: MetaFactoryTypes = {
         }
       }
     })
+  },
+  soundCreatorV1: {
+    newContractCreatedEvent: 'SoundEditionCreated',
+    creationEventToNftFactory: (event: any, autoApprove: boolean, approved: boolean) => ({
+      id: formatAddress(event.args!.soundEdition),
+      platformId: 'sound',
+      startingBlock: event.blockNumber,
+      contractType: NFTContractTypeName.default,
+      standard: NFTStandard.ERC721,
+      autoApprove,
+      approved: autoApprove,
+      typeMetadata: {
+        overrides: {
+          type: MusicPlatformType['multi-track-multiprint-contract'],
+          artist: {
+            artistId: ethereumArtistId(event.args!.deployer),
+            artistName: 'todo', // get from apitracks
+            avatarUrl: 'todo', // get from apitracks
+            websiteUrl: 'todo', //get from apitracks
+          },
+          extractor: {
+            id: IdExtractorTypes.CONTRACT_ADDRESS,
```

Comment:
todo: may be multiple songs in a contract (eg: album) so should use titleExtractor or something instead of just contract_address